yt/frontends/boxlib/data_structures.py

```diff
@@ -892,7 +890,7 @@ def _parse_header_file(self):
             self.geometry = Geometry(geom_str)
 
         if self.geometry == "cylindrical":
-            dre = self.domain_right_edge
+            dre = self.domain_right_edge.copy()
             dre[2] = 2.0 * np.pi
             self.domain_right_edge = dre
```

Comment:
I don't think I get this change, or why this simple operation is done over 3 lines when a single would suffice ?

Reply:
`self.domain_right_edge` is set to be read-only since #4641, so we can't assign to `self.domain_right_edge[2]` directly. `dre` is a mutable copy, so we can modify it.

Reply:
Oh cool, I forgot about this. Thanks !